LICENSE

```diff
@@ -0,0 +1,19 @@
+*********************************************************************************
+*  BlueBrain C++ Coding Conventions
+*
+*  Copyright (c) 2019, 2020, 2021 Blue Brain Project, EPFL.
```

Comment:
nitpick : not sure about the official BBP policies but few weeks ago there were some nitpicks in coreneuron license file : https://github.com/BlueBrain/CoreNeuron/commit/28639a74ba7b91bab6afdef437349f26a601766c. In case it should be updated here.